src/graphql/schema/collection/createCollectionGqlType.ts

```diff
@@ -93,11 +95,13 @@ export default function createCollectionGqlType<TValue> (
               ],
               // We use the config when creating a connection field to inject
               // a condition that limits what we select from the paginator.
-              getPaginatorInput: (headValue: TValue, args: { condition?: { [key: string]: mixed } }) =>
+              getPaginatorInput: (aliasIdentifier: mixed, args: { condition?: { [key: string]: mixed } }) =>
                 conditionHelpers.and(
-                  relation.getTailConditionFromHeadValue!(headValue),
+                  relation.getTailConditionFromHeadAlias!(aliasIdentifier),
```

Comment:
The place to handle it would be here: https://github.com/calebmer/postgraphql/pull/342/files#diff-44b7c98267d37271bb1a324470ae1767R50 - but you're correct - it needs handling